### PR TITLE
Simple objects no longer iterate over all sub populations

### PR DESCRIPTION
**Pull request by itaitzruia4**
Handles #65 
(under the assumption that Simple objects assume 1 sub population)

### CODE REVIEW

**Comment by itaitzruia4** [comment]
should be merged after #104

**Comment by itaitzruia4** [comment]
Closing and reopening to run checks

File: eckity/evaluators/simple_population_evaluator.py [hunk 1]
```diff
@@ -7,52 +7,59 @@
...
+        population:
+                the population of the evolutionary experiment
+
+        Returns
+        -------
+        individual
+                the individual with the best fitness of the given individuals
+        """
+        super()._evaluate(population)
+        sub_population = population.sub_populations[0]
```
Comment (achiyae) [inline eckity/evaluators/simple_population_evaluator.py:38]: add assert before that there is only one sub population

**Review by achiyae — CHANGES_REQUESTED** [review]

File: eckity/evaluators/simple_population_evaluator.py (hunk 1 above)
Comment (itaitzruia4) [reply eckity/evaluators/simple_population_evaluator.py:45]: done, also added a check in `SimpleBreeder`

**Pull request merged**